core/service.go

```diff
@@ -125,7 +125,7 @@ func NewService(cfg *Config) (*Service, error) {
 		if err != nil {
 			log.Error("[core] could not start babe session", "error", err)
 			srv.isBabeAuthority = false
-			return nil, nil
+			return srv, nil
```

Comment:
this function gets called from configcmd.go makeNode, so if the error was returned here, the entire node would shutdown if the babe session fails to start.

it depends on what we want to node functionality to do, if we run a node but can't start babe, should the node shutdown or should it just log and error?

Reply:
logging the error and continuing to start the node would be ok with me for now

Reply:
well if the node is configured as a authority node and it fails to start, that is a severe and should be the error should be propagated. Otherwise its inconsistent.

Reply:
ok, upon further thought, I agree and I think it should probably exit. I will update it in my other PR